content/ros/actions/presentation/ros_actions.tex

```diff
@@ -0,0 +1,436 @@
+        
+        \item Popular case example: sending a pose to the robot, where the action server is responsible for controlling the robot to reach the goal pose.
+        
+        
+    \end{enumerate}
+\end{frame}
+
+
+
+\begin{frame}{Recap}
+    \framesubtitle{ROS Concepts}
+    
+    Concepts related to ROS computation graph:
+    
+    \begin{enumerate}
+        \item \sout{Nodes.}  \checkmark
+        \item \sout{Topics.} \checkmark
+        \item \sout{Messages.} \checkmark
+        \item \sout{Master.} \checkmark
+        \item \sout{Services}. \checkmark
+        \item Actions
+        \item \sout{Parameter Server.} \checkmark
+        \item Bags.
+    \end{enumerate}
+\end{frame}
+
+\section{ROS Actions}
+
+\begin{frame}{ROS Actions}
+
+    {\huge Actions:}
+    \vspace{0.2cm}
+    \begin{itemize}
+        \item Communication happens between two nodes, the action \textbf{server} node, and the action \textbf{client} node.
+        
+        \vspace{2mm}
+        
+        \item A client node sends a \textbf{goal} to the action server. The client does NOT have to wait for the response.
+        
+        \vspace{2mm}
+        
+        \item The server node executes the action, during execution it can optionally send feedback messages to the client.
+        
+    \end{itemize}  
+\end{frame}
+
+
+\begin{frame}{ROS Actions}
+    
+    {\huge Actions:}
+    \vspace{0.2cm}
+    \begin{itemize}
+
+        
+        \item Meanwhile, the client can receive \textbf{feedback} messages from the server, it can also check on the \textbf{status} of the current goal, or can even ask the server to \textbf{cancel} the action.
+        
+        \vspace{2mm}
+        
+        \item Once the server has finished executing the action, it sends a \textbf{result} message back to the client.
+    \end{itemize}  
+\end{frame}
+
+
+\begin{frame}{ROS Actions}
+ \centering
+ \includegraphics[width =0.75\linewidth]{figures/actions.eps}
+  
+  \vspace{5mm}
+ \tiny{\textit{extracted and edited from:       
+   \href{http://wiki.ros.org/actionlib/DetailedDescription}{http://wiki.ros.org/actionlib/DetailedDescription}  }}                                                         
+\end{frame} 
+
+
+
+\begin{frame}{ROS Actions}
+    
+    {\huge Actions:}
+    \vspace{0.2cm}
+    \begin{itemize}
+        
+        
+        \item ROS actions are built on top of ROS messages, they are not part of {\ttfamily \colorbox{gray!30!white}{\href{https://docs.ros.org/kinetic/api/rospy/html/}{rospy}}}. 
+        
+        \vspace{3mm}
+        
+        \item They are provided in the {\ttfamily \colorbox{gray!30!white}{actionlib}} ROS stack which is installed by default when you install ROS.
+        
+        \vspace{3mm}
+        
+        \item There are no command-line tools to introspect ROS actions. Example: \\ \vspace{2mm}  there is no {\ttfamily \colorbox{gray!30!white}{rosaction list}} command!
```

Comment:
But one can use `rostopic` for inspection since actions expose topics; please mention this as a subitem.

Reply:
Done.
(it was written in the README, but it should be here as well :+1: )